lua/ollama.lua

```diff
@@ -0,0 +1,76 @@
+local utils = require 'utils'
+
+local Module = {}
+
+Module.name = 'ollama'
+
+Module.default_model = 'llama3.2:3b'
+
+Module.models = {
+  Module.default_model,
+  'deepseek-coder:1.3b',
+  'tinydolphin:latest',
+  'qwen2.5:7b',
+}
+
+local system_prompt = [[
+Return only the command to be executed as a raw string, no string delimiters
+wrapping it, no yapping, no markdown, no fenced code blocks, what you return
+will be passed to vim directly.
```

Comment:
plea

This is anecdotal based on a couple of tests, the following will prevent the model from returning multiple lines:

```suggestion
Return only the command to be executed as a raw string, no string delimiters
wrapping it, no yapping, no markdown, no fenced code blocks, one line, what
you return will be passed to vim directly.
```